### DISCUSSION TITLE
Should Android (13+) Notification Permissions been requested by calling `requestPermission()`?

### DISCUSSION THREAD

**Discussion by supoved** [invertase/react-native-firebase#7177] [Q&A]
As far as my testing goes calling `firebaseMessaging.requestPermission()` doenst do anything on android including Android 13 where notification permission is now required same as it is on iOS.

I cant find any issue of people complaining or requesting it, which is odd as I doubt people only target old android and ios. 

What am I missing?

**Comment by nl-danish** [comment]
What i did, was implemented another code for asking permissions for Android 13+ devices,

I believe it is something that needs to be asked separated other than firebaseMessaging.requestPermission().

**Comment by jgarplind** [comment]
I ran across this answer some time ago: https://github.com/invertase/react-native-firebase/issues/6283#issuecomment-1401974549

Basically, it is recommended to use react-native-permissions to handle permissions instead.
